matRad_fluenceOptimization.m

```diff
@@ -247,7 +250,8 @@
 
 %Give scenarios used for optimization
 backProjection.scenarios    = ixForOpt;
-backProjection.scenarioProb = pln.multScen.scenProb;
+backProjection.scenarioProb = repmat(pln.multScen.scenProb, pln.multScen.numOfCtScen,1);
```

Comment:
I don't understand what motivates the change in this line. It is not used anywhere else in this PR? Was it wrong before?

Reply:
Sorry this slipped in here, I was not planning to include it. Shall I commit a revert?